net-misc/trackma/metadata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<longdescription>
+	Open multi-site list manager for Unix-like systems. (ex-wMAL)
+</longdescription>
+<use>
+	<flag name="cli">Enable cli version</flag>
+	<flag name="urwid">Enable curses version</flag>
```

Comment:
Since it's just a wrapper over ncurses, wouldn't it be better to use USE=ncurses?